### ISSUE TITLE
Color Scheme for vueifyjs

### ISSUE THREAD

**Issue by mneuschaefer** [Squarelet/squarelet#15]
Which default colors to use for our menu and canvas? 

As we're transitioning to https://vuetifyjs.com , we need a color scheme as a basis to design menu elements. A good combination should be fresh, recognizable, [WCAG-compliant](https://www.w3.org/WAI/standards-guidelines/wcag/) and use the colors from the palette of the vuetify theme generator: https://vuetifyjs.com/en/theme-generator

Which colors would work well for squarelet ui? Please post your examples with pictures.

**Comment by mneuschaefer** [comment]
a simple interface with dark blue seems to work well for a start (sparse use of colors to highlight user content), it can be user-customized later

![image](https://user-images.githubusercontent.com/11058806/49033218-d7af6600-f1ae-11e8-9e1a-b75a500cee91.png)



![image](https://user-images.githubusercontent.com/11058806/49033229-dbdb8380-f1ae-11e8-9e0c-51ac27bfc258.png)



https://codepen.io/markusn/pen/eQrLgb
